testing/jupyterlab/README.md

```diff
@@ -4,38 +4,63 @@
 
 ## Running the accessibility tests ✅
 
 You can run tests locally on your machine, or remotely in the cloud.
 
-At the time of this writing, we have configured the tests to only run with
-Chromium, but you can
-[extend these tests to cover other browsers](https://github.com/MarcusFelling/demo.playwright/blob/main/accessibility/playwright.config.ts)
-by modifying the [`playwright.config.ts`](testing/jupyterlab/playwright.config.ts) file.
+> **Note**
+> At the time of this writing, we have configured the tests to only run with Chromium,
+> but you can [extend these tests to cover other browsers](https://github.com/MarcusFelling/demo.playwright/blob/main/accessibility/playwright.config.ts)
+> by modifying the [`playwright.config.ts`](testing/jupyterlab/playwright.config.ts) file.
 
 ### Running the tests on GitHub ☁️
 
 1. Go to the [JupyterLab accessibility testing
    workflow](https://github.com/Quansight-Labs/jupyter-a11y-testing/actions/workflows/accessibility-test-jupyterlab.yml).
 
+   > **Note:** this link will redirect you to the GitHub Actions UI.
+
 2. Click the "Run Workflow" button. This should open a dropdown form.
 
 3. In the dropdown form, enter:
 
-   1.  the JupyterLab repo or fork that you want to test
-   2.  the SHA of the commit in the JupyterLab repo that you want to test
-   3.  (optional) the subset of tests that you want to run (e.g., enter
-       `regression` to run only the accessibility regression tests).
+   1. The JupyterLab repo or fork that you want to test
+   2. The SHA of the commit in the JupyterLab repo that you want to test
+   3. (optional) The subset of tests that you want to run (e.g., enter `regression` to run only the accessibility regression tests).
+
+#### Inspecting the test results
+
+Once the accessibility tests have been completed you can use the following to inspect the test results:
+
+- [**Annotations**](https://playwright.dev/docs/test-reporters#github-actions-annotations): this will appear directly in the GitHub actions UI.
+- **Markdown summary**: scroll to the bottom of the GitHub actions UI, where you can find this summary. From here you can use the links to read the manual test scripts for the corresponding tests.
+- **Zipped `json` and `HTML` reports**: these can be downloaded from the GitHub actions UI. The `json` report can be used to generate a custom report using the [Playwright Reporter](https://playwright.dev/docs/test-reporters#custom-reporter). The `HTML` report can be opened in a browser to view the test results.
+
+Follow these steps to download and read the reports locally:
+
+1. Scroll to the bottom of the GitHub actions UI and right-click on the test results' artifact.
+2. Unzip the downloaded file. Ideally in a folder where you have `npm` and `playwright` installed. For example, in the `testing/jupyterlab` directory.
+   (Tip: see the instructions in [Running the tests locally](#running-the-tests-locally-) to install the corresponding dependencies)
```

Comment:
The link on this line looks funny with that dangling dash. Are you sure it's right?

Reply:
yes it is correct - it's because of the emoji at the end of the heading that it ends in a `-`